### ISSUE TITLE
strange line breaks / indentation

### ISSUE THREAD

**Issue by JoernBerkefeld** [fib-at-isobar/beautyAmp#13]
I've been trying this with the following code (ok, original obviously had different URLs but I then tested with this as well):

**Expected formatting:**
![image](https://user-images.githubusercontent.com/1917227/123592193-30bf5a80-d7ed-11eb-9b01-747f02bb9a40.png)

**Actual formatting:**
(--> look at how lines 10+11 get merged and the indentation of line 10 changes)
![image](https://user-images.githubusercontent.com/1917227/123592152-22713e80-d7ed-11eb-90b6-cac835aeeda3.png)

in some variants it even did the weird ELSEIF positioning in 2 occasions. 

**Source Code for testing:**

```
%%[
    IF @language == "en" THEN
        /* English */
        SET @language = "EN"
        SET @link = "https://www.mydemo.com/en/some/path/down/the/rabbit/hole.html"
    ELSEIF @language == "fr" THEN
        /* French */
        SET @language = "FR"
        SET @link = "https://www.mydemo.com/en/some/path/down/the/rabbit/hole.html"
    ELSEIF @language == "de" THEN
        /* German */
        SET @language = "DE"
        SET @link = "https://www.mydemo.com/en/some/path/down/the/rabbit/hole.html"
    ELSEIF @language == "it" THEN
        /* Italian */
        SET @language = "IT"
        SET @link = "https://www.mydemo.com/en/some/path/down/the/rabbit/hole.html"
    ENDIF
]%%
```

**Comment by fib-at-isobar** [comment]
Hello @JoernBerkefeld ,
Thanks for telling me! I will work on it ASAP.

Regards,
Filip

**Comment by fib-at-isobar** [comment]
Hello @JoernBerkefeld ,
Seems I have found the issue. I am currently testing this fix, and will release it soon.
Regards,
Filip

**Comment by fib-at-isobar** [comment]
Hello,
fixed with new version 0.4.0 - enjoy!

**Comment by JoernBerkefeld** [comment]
very cool - good job :)
